packages/server/graphql/mutations/helpers/handleCompletedStage.ts

```diff
@@ -111,15 +112,16 @@ const handleCompletedRetrospectiveStage = async (
     }))
     await Promise.all([
       insertDiscussions(discussions),
-      addAIGeneratedContentToThreads(discussPhaseStages, meetingId, teamId, dataLoader)
+      addAIGeneratedContentToThreads(discussPhaseStages, meetingId, teamId, dataLoader),
+      generateRelatedDiscussions(meeting, teamId, dataLoader)
```

Comment:
Q: is the embedding created here the same as the embedding created when the meeting ends?